Makefile

```diff
@@ -101,4 +101,15 @@ hana-database-plugin:
 mongodb-database-plugin:
 	@CGO_ENABLED=0 go build -o bin/mongodb-database-plugin ./plugins/database/mongodb/mongodb-database-plugin
 
+govendor:
+	go get github.com/kardianos/govendor
+
+use-mesosphere-zookeeper-fork: govendor
+	find . \
+		-name '*.go' \
+		-not -path '*/vendor/*' \
+		-not -path '*/.git/*' \
+		-exec sed -i '' -e 's/samuel\/go-zookeeper/mesosphere\/go-zookeeper/' {} \;
```

Comment:
We can also use `sed -i '' -e 's#samuel/go-zookeeper#mesosphere/go-zookeeper#'`

Reply:
Thanks for the suggestion, I've update the code.